### PR TITLE
#465 blog header pop up style 

### PR DESCRIPTION
**Pull request by kartikhsr**
#465 i have fixed the issue of blog header pop up style

### CODE REVIEW

**Comment by m1guelpf** [comment]
@kartikhsr Could you share a preview link?

**Comment by codein-dwij** [comment]
Please fix the merge conflicts too.
Can I know what happened to travis?

**Comment by kartikhsr** [comment]
@m1guelpf 
screenshot from 2016-12-12 16-58-51
i have also fixed the MBDyn logo designs header pop up style as same as done for blog header 
screenshot from 2016-12-12 17-13-57

**Comment by rhemon** [comment]
@kartikhsr  give us your github page link and could you also resolve the merge confiicts?

**Comment by kartikhsr** [comment]
@rhemon https://github.com/kartikhsr/gci16.fossasia.org, can you give me idea about merge conflicts...then i will surely try

**Comment by oanarosca** [comment]
You can find help in the contributing.md file, check that out! And if you need live help, join us on slack, we'd be happy to walk you through this :)

**Review by daminisatya — DISMISSED** [review]
Good job but Please resolve merge conflicts! :)

**Comment by kartikhsr** [comment]
@daminisatya i am trying to solve merge issue

**Comment by hoangvanthien** [comment]
@kartikhsr How's it going, do you need any help? :) You should remove any part wrapped by `<<<<<` and `=====`

**Comment by kartikhsr** [comment]
@hoangvanthien Actually i cant make a pull request that could be the reason ... !!

**Comment by m1guelpf** [comment]
@kartikhsr You have made a pull request. This is a pull request.

**Comment by kartikhsr** [comment]
@hoangvanthien thank you

**Comment by hoangvanthien** [comment]
Do not merge yet. 
@kartikhsr What you need to do now is squashing commits and returning the current index.html. Give me your slack account, I will guide you through.

**Comment by m1guelpf** [comment]
@kartikhsr Looks good to me, but I will not merge it as I don't know the "situation". It's better to wait for a mentor that knows all about this task to review it.
Also, could you share the preview link? Usually it is [yourusername].github.io/gci16.fossasia.org
@hoangvanthien why?

**Comment by kartikhsr** [comment]
@hoangvanthien slack-username : kartikhsr85

**Comment by myfrom** [comment]
@hoangvanthien isn't there an option to squash commits when merging PR?

**Comment by m1guelpf** [comment]
@Witus13 Yep, there is one.

**Comment by niccokunzmann** [comment]
@m1guelpf Hi, I propose: merging pull-requests and reviewing tasks are to separate things. The one can be done without the other. In order to keep the contributions small and merged fast, with higher collaboration, I think, as a maintainer, you can merge if you think it is fine.

I also believe this because I think the evaluation process of the competition is conflicting with the joy of collaboration. In the end: to me and I believe to many mentors it is important that you have fun when working on these projects, joy and fulfillment and purpose.

If a maintainer merges, someone who knows about the issue closes it, I it is fine with me. For the evaluation, I will read through it to make sure it is not a fake. After all: I do not know, what the issue meant to the person opening it if it is not me. So I can hardly decide if it should be closed.

@m1guelpf If you have doubts or fear, please tell me, so we can talk about it and remove them:)

**Comment by niccokunzmann** [comment]
Short form: @m1guelpf you can merge when you like. If you fear to do something wrong, this is an indicator for me that we should talk to remove that fear.
(Maybe I was also off-topic now since I did not read much)

**Comment by codein-dwij** [comment]
@kartikhsr Are you on slack? Please tell me your username if you need any help resolving merge conflicts

**Comment by kartikhsr** [comment]
@codein-dwij username:kartikhsr85

**Comment by codein-dwij** [comment]
Great Merge Conflicts are resolved :)

File: index.html
Comment (hoangvanthien) [inline index.html:292]: Why did you make change here?

**Pull request merged**
